flux-plugin/src/flux-image-automation/image-automations.tsx

```diff
@@ -0,0 +1,270 @@
+import {
+  DateLabel,
+  Link,
+  SectionBox,
+  ShowHideLabel,
+  Table,
+} from '@kinvolk/headlamp-plugin/lib/components/common';
+import { K8s } from '@kinvolk/headlamp-plugin/lib';
+import { apiFactory } from '@kinvolk/headlamp-plugin/lib/ApiProxy';
+import React from 'react';
+import { KubeObject } from '@kinvolk/headlamp-plugin/lib/lib/k8s/cluster';
+import CheckIfFluxInstalled from '../checkflux';
+
+export default function ImageAutomation() {
+  const CRD = K8s.ResourceClasses.CustomResourceDefinition;
+  CRD.apiEndpoint = apiFactory(
+    ['apiextensions.k8s.io', 'v1', 'customresourcedefinitions'],
+    ['apiextensions.k8s.io', 'v1beta1', 'customresourcedefinitions'],
+    ['apiextensions.k8s.io', 'v1beta2', 'customresourcedefinitions']
+  );
```

Comment:
Would it work to simply add the 3rd value if there's no v1beta2 in the apiEndpoint (this should be checked only once)? I am mentioning this because we will add this new API version and maybe more in the future, and then we have to update the plugin for it to work with versions past these 3.

Reply:
updated